mix.exs

```diff
@@ -41,7 +41,10 @@ defmodule Thrift.Mixfile do
      package: package(),
 
      # Dialyzer
-     dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:ex_unit, :mix, :public_key, :ssl]],
+     dialyzer: [
+       plt_add_deps: :transitive,
+       plt_add_apps: [:mix],
+       ignore_warnings: ".dialyzer.warnings"],
```

Comment:
If I'm being super nitpicky, `.dialyzer.warnings` doesn't convey "these are warnings that will be ignored".  What about something like`.dialyzerignore` (in the vein of `.gitignore`, etc)?